src/utils/series.ts

```diff
@@ -4,8 +4,10 @@ import { getEpisodes, getSeasonWithEpisodes } from '#src/services/api.service';
 /**
  * Get an array of options for a season filter
  */
-export const getFiltersFromSeries = (series: Series | undefined): string[] => {
-  return series && 'seasons' in series ? series.seasons.map((season) => String(season.season_number)) : [];
+export const getFiltersFromSeries = (series: Series | undefined): { label: string; value: string }[] => {
+  return series && 'seasons' in series
+    ? series.seasons.map((season) => ({ label: season.season_title || String(season.season_number), value: String(season.season_number) }))
```

Comment:
Shouldn't we include the seasons prefix here since it was removed from the filterValuePrefix prop?

Reply:
Good point... will season_title ever end up empty unless they try really hard to do so? If that's the case, then I'm not worried about this I guess.

Reply:
It can be empty now in UI, but we can make it `required` field. Looks like we should.

Reply:
Let's leave it as optional and emptyable for now. Some customers may prefer the language support of unnamed seasons instead of customizing titles themselves. In this case, we should use `season_number_filter_template` here.

Honestly, I can't think of an example where a series I've watched has meaningful names for seasons. Maybe for e-learning type courses it would make sense, but normal content not so much.

Reply:
I saw that sometimes customers use years for season titles. I think e-learning is also a valid example.

Reply:
OK, I'm fine with however it's setup now. We can always iterate on it.